### PR TITLE
Add Windows support

### PR DESCRIPTION
**Pull request by spyric**
This PR add Windows support. 

I have no chance to test it on linux or mac, but hope It should work

### CODE REVIEW

File: src/Console/DuskCommand.php [hunk 1]
```diff
@@ -45,13 +46,29 @@ public function handle()
...
+        );
+    }
+
+    private function getPathToPhpUnit()
+    {
+        if (PHP_OS === 'WINNT') {
+            return base_path('vendor\bin\phpunit.bat');
+        }
+
+        return 'php vendor/bin/phpunit';
```
Comment (svenluijten) [inline src/Console/DuskCommand.php]: Should this not also use the `base_path()` function?

Comment (spyric) [inline src/Console/DuskCommand.php]: It was from first version. I tried to keep all that is not connected with windows unchanged. If you have linux or macOS can you check if it work with `base_path()`?

**Comment by KKSzymanowski** [comment]
Works for me. 👍

**Comment by alex-LE** [comment]
For me too. Windows 10 64 Bit

File: src/SupportsChrome.php [hunk 2]
```diff
@@ -42,4 +39,22 @@ public static function stopChromeDriver()
             static::$chromeProcess->stop();
         }
     }
+
+    protected static function initChromeDriver(){
+        static::$chromeProcess = (new ProcessBuilder())
+            ->setPrefix(realpath(__DIR__.'/../bin/chromedriver-' . static::getOSSuffix() . '.exe'))
```
Comment (ipalaus) [inline src/SupportsChrome.php]: That `.exe` extension should happen only when is `WINNT`.

Comment (spyric) [inline src/SupportsChrome.php]: Good catch! I have moved it to `getOSSuffix()` so it should work for non WINNT platforms

**Comment by DeeJaVu** [comment]
does not work on windows 2.0 pls help

**Comment by KKSzymanowski** [comment]
@DeeJaVu What's Windows 2.0?

**Comment by DeeJaVu** [comment]
@KKSzymanowski Windows from 1987

**Comment by KKSzymanowski** [comment]
giphy 1

**Comment by taylorotwell** [comment]
Breaks on my Mac: sh: php vendor/bin/phpunit: No such file or directory ... Worked before.

**Comment by taylorotwell** [comment]
Also please revert all unrelated spacing changes that were made.

**Comment by KKSzymanowski** [comment]
I have no way to test it on a Mac. Can you give us more info?

**Comment by spyric** [comment]
@taylorotwell Can you check it now?

**Comment by spyric** [comment]
Have checked on Ubuntu 16.04 x64. All work fine

**Comment by KKSzymanowski** [comment]
I have managed to install Mac OS X and I can confirm that the newest version works.

**Comment by spyric** [comment]
@taylorotwell have you tested it? Is it works for you now?

**Comment by KKSzymanowski** [comment]
I too am curious. I'd like to see this merged as soon as possible. Can we help you with anything?

**Comment by mrluissan** [comment]
It works on Windows 10

**Comment by alex-LE** [comment]
@spyric could you please fix the conflict? It might be "better" for @taylorotwell to merge it ;-)

**Comment by spyric** [comment]
@alex-LE Conflict is resolved

**Comment by rawzone** [comment]
Can confirm that this made dusk work on my Arch Linux desktop as well.

**Comment by lasselehtinen** [comment]
Confirmed to work on Windows 10.

What guys you are getting as execution time? Because I am getting closer to six seconds just with the provided example. I was watching the Laracast video about Dusk and Jeffrey has something like less than one second.

```bash
PHPUnit 5.7.8 by Sebastian Bergmann and contributors.

.                                                                   1 / 1 (100%)

Time: 5.72 seconds, Memory: 8.00MB

OK (1 test, 1 assertion)
```

**Comment by taylorotwell** [comment]
I still don't understand why unrelated spacing changes are being made. It halts the inclusion of this PR.

**Comment by taylorotwell** [comment]
The rest of the Laravel framework has doc-blocks on every method.

**Comment by taylorotwell** [comment]
Cleaned up all the spacing and other formatting issues. Thanks for the help with Windows debugging!

**Comment by jcwirko** [comment]
I need help. I have laravel 5.4, windows 10 with virtual box and homestead and I dont know how to change this files in my project. 

In the cmd if I execute php artisan dusk I get:  
\vendor\bin\phpunit.bat"' is not recognized as an internal or external command, operable program or batch file.

File: src/SupportsChrome.php [hunk 3]
```diff
@@ -18,17 +19,15 @@
      */
     public static function startChromeDriver()
     {
-        if (PHP_OS === 'Darwin') {
```
Comment (mivashchuk) [inline src/SupportsChrome.php]: `

**Comment by pildit** [comment]
Hi! how can I make it work on Windows 7 ? 
I get this error 'TTY mode is not supported on Windows platform' when i run php artisan dusk
Laravel 5.4
Thank you !

**Comment by deleugpn** [comment]
That's just a warning.

**Comment by pildit** [comment]
Well, maybe it's just a warning, but why `php artisan dusk` not displays antyhing besides this ?
looks like a Symfony Runtime Exception

**Comment by wangyaoxu** [comment]
Works for me on Windows 7. :+1:

**Comment by pildit** [comment]
Works for me too, I copied the files from github

**Pull request merged**